src/lang/en.json

```diff
@@ -1098,5 +1102,9 @@
     "Phone numbers": "Phone numbers",
     "Sender name": "Sender name",
     "smsplanetNeedToApproveName": "Needs to be approved in the client panel",
-    "Disable URL in Notification": "Disable URL in Notification"
+    "Disable URL in Notification": "Disable URL in Notification",
+    "Add Another Tag": "Add Another Tag",
+    "Staged Tags for Batch Add": "Staged Tags for Batch Add",
```

Comment:
also please make a pass that all translations are listed in this file.
If a key (like `add tags`) is not here, it is not translatable into other languages by our kickass translators 😉